src/main/java/com/prowidesoftware/swift/constraints/BicConstraint.java

```diff
@@ -22,8 +22,8 @@
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
 
-import javax.validation.Constraint;
-import javax.validation.Payload;
+import jakarta.validation.Constraint;
```

Comment:
See comment about import order above.

Reply:
We normally do organize import with the IDE formatter and leave the order that is produced by it.

Reply:
> We normally do organize import with the IDE formatter and leave the order that is produced by it.

The import order is defined in Eclipse config file `org.eclipse.jdt.ui.prefs` (see above) This file treated `java|javax` imports differently. `jakarta` is like any other package name.